ape_foundry/provider.py

```diff
@@ -673,7 +579,12 @@ def _handle_execution_reverted(
                 # Unlikely scenario where a transaction is on the error even though a receipt
                 # exists.
                 if isinstance(enriched.txn, TransactionAPI) and enriched.txn.receipt:
-                    enriched.txn.receipt.show_trace()
+                    try:
+                        enriched.txn.receipt.show_trace()
+                    except Exception:
+                        # TODO: Fix this in Ape
```

Comment:
basically empty reverts from foundry and causing problem in Ape, nothing really broken in the plugin but i have to fix this in ape